rezolvari probleme.js

```diff
@@ -0,0 +1,71 @@
+//problema 1
+function SumVect(vect)
+{
+    var sum = 0;
+
+    for(i=0; i < vect.length; i++)
+    {
+        sum += vect[i];
+    }
+
+    return sum;
+}
+
+//problema 2
```

Comment:
"Good code should be self explanatory"

This comments are wrong because:
- you should have renamed the function
- they are in Romanian, there is no requirement written in Romanian.